config/oidc/configmap.yaml

```diff
@@ -0,0 +1,21 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: oidc-config
+data:
+  auth-svr-config.yaml: |
+    address: ":9443"
+    issuerURL: "http://dex.auth.app.192.168.10.134.nip.io:9080"
+    redirectURL: "https://echo.oidc.app.local:9443"
```

Comment:
I think this is okay for the first version, but it will need a TODO issue. I'll log one and update here. Once I do, @robinfoe, can you please drop a comment in here and in the docs for the command?

Reply:
This is #17.